cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,107 @@
 describe('Web Tables page', () => {
+  const user = {
+    firstName: 'angel',
+    lastName: 'sen',
+    email: 'angelsen@gmail.com',
+    age: '10',
+    salary: '10000',
+    department: '123'
+  };
+
+  const newUser = {
+    firstName: 'Angelina',
+    lastName: 'Sen',
+    email: 'angelinasen@gmail.com',
+    age: '120',
+    salary: '100500',
+    department: '231'
+  };
+  beforeEach(() =>{
+    cy.visit('https://demoqa.com/webtables');
+  });
+
   it('', () => {
```

Comment:
The test suite seems to be missing a test case for checking search by all column values. Please add